tests/system/test_install.py

```diff
@@ -7,18 +7,18 @@
  C: a commit after a tag, clean tree
  D: a commit after a tag, dirty tree
 
-The tests written in this file use pytest-virtualenv to achieve isolation.
+The tests written in this file use venv to achieve isolation.
```

Comment:
Is this right? I thought we are now still using pytest-virtualenv but wrapping it with some helpers in `Venv`?

Reply:
Sorry, I did a quick merge between the branches and corrected the errors, but I forgot to come back correcting the documentation.

You are completely right. I have to change this comment.